Fix typo in ILL_ILLOPN definition.

===================================================================
RCS file: /cvs/glibc/libc/bits/siginfo.h,v
retrieving revision 1.6
retrieving revision 1.7
diff -u -r1.6 -r1.7
--- libc/sysdeps/generic/bits/siginfo.h	2001/07/06 04:55:50	1.6
+++ libc/sysdeps/generic/bits/siginfo.h	2006/04/01 19:18:59	1.7
@@ -76,7 +76,7 @@
 {
   ILL_ILLOPC = 1,		/* Illegal opcode.  */
 # define ILL_ILLOPC	ILL_ILLOPC
-  ILL_ILL_OPN,			/* Illegal operand.  */
+  ILL_ILLOPN,			/* Illegal operand.  */
 # define ILL_ILLOPN	ILL_ILLOPN
   ILL_ILLADR,			/* Illegal addressing mode.  */
 # define ILL_ILLADR	ILL_ILLADR
